db/migrate/20190820095253_copy_enrichments_to_next_cycle.rb

```diff
@@ -0,0 +1,25 @@
+class CopyEnrichmentsToNextCycle < ActiveRecord::Migration[5.2]
+  def up
+    next_providers = RecruitmentCycle.next_recruitment_cycle.providers.all
+    rollover_service = ProviderEnrichments::RolloverEnrichmentToProviderService.new
+
+    current_providers_with_enrichments.each do |provider|
+      next if provider.enrichments.empty?
```

Comment:
Not a blocker since this is a throwaway, but if you end up changing anything else here this check is now redundant because you now do it in `current_providers_with_enrichments`.